controller/webapis/oidc-api.go

```diff
@@ -86,7 +86,7 @@ func (h OidcApiHandler) RootPath() string {
 }
 
 func (h OidcApiHandler) IsHandler(r *http.Request) bool {
-	return strings.HasPrefix(r.URL.Path, h.RootPath()) || strings.HasPrefix(r.URL.Path, oidc_auth.WellKnownOidcConfiguration)
+	return strings.HasPrefix(r.URL.Path, h.RootPath()) || r.URL.Path == oidc_auth.WellKnownOidcConfiguration
```

Comment:
should this be and? what if the edge/v1/client|management is prefixed? should that be allowed?

Reply:
No.

This is checking if the path is `/oidc` or `/.well-known/opendid-configuration`